.bazelrc

```diff
@@ -17,7 +17,6 @@
 
 try-import ./.bazel-remote-cache.rc
 
-build --incompatible_strict_action_env --java_language_version=11 --java_runtime_version=remotejdk_11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false
-run --incompatible_strict_action_env
-test --incompatible_strict_action_env --test_env=PATH
-
+build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false --java_runtime_version=remotejdk_11
+run --incompatible_strict_action_env --java_runtime_version=remotejdk_11
+test --incompatible_strict_action_env --test_env=PATH --java_runtime_version=remotejdk_11
```

Comment:
is there a 'common' block?

Reply:
try putting it first. Move `--incompatible_strict_action_env` there as well if it works, then delete `run` since it'l be empty

Reply:
`Inherited 'common' options: --java_runtime_version=remotejdk_11 --incompatible_strict_action_env
ERROR: --java_runtime_version=remotejdk_11 :: Unrecognized option: --java_runtime_version=remotejdk_11`